site/_posts/2019-09-20-Velero-v1-1-on-PKS.md

```diff
@@ -0,0 +1,350 @@
+pks-system    fluent-bit       3         3         3       3            3           <none>          5d3h
+pks-system    telegraf         3         3         3       3            3           <none>          5d3h
+velero        restic           3         3         0       3            0           <none>
+```
+
+Next, edit the DaemonSet and change the hostPath setting. The before and after edits are shown below.
+
+```bash
+$ kubectl edit ds restic -n velero
+
+# Before
+      volumes:
+      - hostPath:
+          path: /var/lib/kubelet/pods
+          type: ""
+        name: host-pods
+
+#After
+      volumes:
+      - hostPath:
+          path: /var/vcap/data/kubelet/pods
+          type: ""
+        name: host-pods
+
+daemonset.extensions/restic edited
+```
+
+## Deploy a sample application to backup
+
+Velero provides a sample `nginx` application for backup testing. This `nginx` deployment assumes the presence of a LoadBalancer for its Service. PKS supports NSX-T integration, so if this is configured, NSX-T will provide this service for you. If you do not have a Load Balancer as part of your Container Network Interface (CNI), there are some easily configuration ones available to get your started. One example is MetalLb, available [here](https://metallb.universe.tf/).
+
+* Note: This application is stateless. It does not create any Persistent Volumes, thus the restic driver is not utilizied as part of this example. To test whether restic is working correctly, you will need to backup a stateful application that is using Persistent Volumes.
+
+To deploy the sample nginx application, run the following command:
+
+```bash
+$ kubectl apply -f examples/nginx-app/base.yaml
+namespace/nginx-example created
+deployment.apps/nginx-deployment created
+service/my-nginx created
+```
+
+Check that the deployment was successful using the following commands:
+
+```bash
+$ kubectl get ns
+NAME                  STATUS   AGE
+cassandra             Active   23h
+default               Active   5d3h
+kube-public           Active   5d3h
+kube-system           Active   5d3h
+nginx-example         Active   4s
+pks-system            Active   5d3h
+velero                Active   9m40s
+wavefront-collector   Active   24h
+```
+
+```bash
+$ kubectl get deployments --namespace=nginx-example
+NAME               READY   UP-TO-DATE   AVAILABLE   AGE
+nginx-deployment   2/2     2            2           20s
+```
+
+```bash
+$ kubectl get svc --namespace=nginx-example
+NAME       TYPE           CLUSTER-IP       EXTERNAL-IP                 PORT(S)        AGE
+my-nginx   LoadBalancer   10.100.200.147   100.64.0.1,192.168.191.70   80:30942/TCP   32s
+```
+
+In this example, a Load Balancer has provided the `nginx` service with an external IP address of 192.168.191.70. If I point a browser to that IP address, I get an nginx landing page identical to that shown below.
+
+![nginx landing page](../img/vsphere-tutorial-icons/nginx.png)
```

Comment:
this image link is also broken - I don't see this file in the PR

Reply:
Good catch - forgot to include the folder when we re-created the PR.
Fixed now (also removed the ../ and referenced /img directly)